src/displayapp/screens/Dice.h

```diff
@@ -0,0 +1,50 @@
+#pragma once
+
+#include "systemtask/SystemTask.h"
+#include "components/motion/MotionController.h"
+#include "components/motor/MotorController.h"
+
+#include <array>
+
+namespace Pinetime::Applications::Screens {
+
+  class Dice : public Screen {
+  public:
+    Dice(DisplayApp* app,
+         Controllers::MotionController& motionController,
+         Controllers::MotorController& motorController,
+         System::SystemTask& systemTask);
+    ~Dice() override;
+    bool Refresh() override;
+    bool OnTouchEvent(TouchEvents event) override;
+
+    void numDiceIncBtnEventHandler(lv_event_t event);
+    void numDiceDecBtnEventHandler(lv_event_t event);
+    void diceSizeIncBtnEventHandler(lv_event_t event);
+    void diceSizeDecBtnEventHandler(lv_event_t event);
+    void rollBtnEventHandler(lv_event_t event);
+
+  private:
+    Pinetime::System::SystemTask& systemTask;
+    Controllers::MotionController& motionController;
+    Controllers::MotorController& motorController;
+
+    static const uint8_t diceSizes[];
```

Comment:
I think You could use `std::array` here

Reply:
I'll switch it over to `std::array` if that would be preferable.

Reply:
I would also suggest adding a constant for the number of dice sizes supported (which is non-optional if using `std::array` anyways). Otherwise it's implicitly defined by the definition of `diceSizes` which can cause issues if it ever changes in the future and the other places where it's currently explicitly repeated (ie. `diceSizeIncBtnEventHandler`) don't get updated to match.

Reply:
@hatmajster I was able to implement this using `std::array` but when I did, the application would fail to load unless I opened a couple other apps before attempting to open the dice app.  I looked around a bit and other places in the code (like `DateTimeController`) use the built-in array for a similar purpose, so I think I'm just going to leave it as-is.

@jonvmey Added a constant for the number of dice sizes in https://github.com/JF002/InfiniTime/pull/565/commits/42b048d648c6f2aad3064e1760771321f0d08e19

Reply:
The constant doesn't completely fix the issue. Still if just the array was changed, the code would break. If you're not going to use std::array, sizeof would probably be better.

Reply:
@Riksu9000 Fixed this in https://github.com/JF002/InfiniTime/pull/565/commits/7c0174284033c67c6e11b61d2829570715e35cb0